SUPPORT.md

```diff
@@ -0,0 +1,44 @@
+# Support Guidelines
+
+We strive to create clear guidelines on communication to the Zarf team to provide a good community experience.
+
+
+## Standard Process
+All questions, and code issues should be a [Github Issue](https://github.com/defenseunicorns/zarf/issues/new/choose) that follows the issue template. 
```

Comment:
Thoughts on adding a hook here for checking out the docs first? Not sure exactly the best way to word it